.github/workflows/test.yml

```diff
@@ -0,0 +1,106 @@
+name: Tests
+
+on:
+  push:
+    branches: [ main, feature/* ]
+  pull_request:
+    branches: [ main ]
```

Comment:
Why do we need an `on` section here?  Shouldn't it just always run like the other workflow?